run_dir/static/js/projects.js

```diff
@@ -325,7 +329,12 @@ function init_listjs(no_items, columns) {
           "paging":false,
           "destroy": true,
           "info":false,
-          "order": [[ 0, "desc" ]]
+          "order": [[ 0, "desc" ]],
+          "stateSave": true,
+          "stateLoadCallback": function (settings) {
```

Comment:
Not sure is `settings` is needed since we aren't really passing it any parameters